bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusThingHandler.java

```diff
@@ -152,7 +162,14 @@ private void initializeThing(ThingStatus bridgeStatus) {
                 if (subAddress != null) {
                     subAddresses[i] = hexToByte(subAddress);
                 } else {
-                    subAddresses[i] = (byte) 0xFF;
+                    if (getConfig().get(propertyKey) != null) {
+                        subAddress = getConfig().get(propertyKey).toString();
+                        if (subAddress != null) {
```

Comment:
It's possible to have a module with a sub-adresse that is not used/configured.
If this is the case, the subadresse is set to 0xFF by the Velbus configuration program.

From the UI, sub-addresses are set by default to 0xFF.
This check is there for users that create things with configuration files. 
In this case it's possible to avoid to inform the references of the sub-addresses. 
It's not a error or miss configuration, they don't use the sub-addresses. 
Here we consider that the sub-address is not used and set it to the default one.

If a sub-address is set to 0xFF, we don't send requests to get informations.